include/swift/Subsystems.h

```diff
@@ -279,12 +281,13 @@ namespace swift {
 
   /// Wrap a serialized module inside a swift AST section in an object file.
   void createSwiftModuleObjectFile(SILModule &SILMod, StringRef Buffer,
-                                   StringRef OutputPath);
+                                   StringRef ModuleName,
+                                   const PrimarySpecificPaths PSPs);
```

Comment:
Stray `const` (or was it meant to be a reference?)

Reply:
This parameter is now a StringRef. Sorry for the interim confusion.